mage_integrations/mage_integrations/destinations/bigquery/__init__.py

```diff
@@ -265,7 +259,7 @@ def process_queries(
                         record_data=record_data,
                         stream=stream,
                         tags=tags,
-                        tries=tries,
```

Comment:
I think the parameter passing also works here. `tries` is passed via kwargs.

Reply:
@wangxiaoyou1993 the signature of the same function changed when calling recursively changed if passing `tries` this way, i.e.,
**Old signature**
```
    def process_queries(
        self,
        query_strings: List[str],
        record_data: List[Dict],
        stream: str,
        tags: Dict = None,
        **kwargs,
    ) 
```
**New signature**
```
    return self.process_queries(
                  query_strings=query_strings,
                  record_data=record_data,
                  stream=stream,
                  tags=tags,
                  tries=tries,
```
`kwargs` may need to be updated to contain the latest `tries` parameter if we indeed want to pass `tries` through `kwargs`.